intera_examples/scripts/go_to_cartesian_pose.py

```diff
@@ -86,6 +86,9 @@ def main():
     parser.add_argument(
         "--rotational_accel", type=float, default=1.57,
         help="The max rotational acceleration of the endpoint (rad/s/s)")
+    parser.add_argument(
+        "-to",  "--timeout", type=float, default=None,
```

Comment:
The short parameter name should only be one character. Since 't' and 'T' is already taken, I would recommend just not having a short parameter name.  Same with the other scripts.

Reply:
Done.